test/src/test/java/org/corfudb/integration/AbstractIntegrationTest.java

```diff
@@ -0,0 +1,147 @@
+package org.corfudb.integration;
+
+import org.apache.commons.io.FileUtils;
+import org.corfudb.AbstractCorfuTest;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+import org.junit.After;
+import org.junit.Before;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.lang.reflect.Field;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.concurrent.Executors;
+
+/**
+ * Created by zlokhandwala on 4/28/17.
+ */
+public class AbstractIntegrationTest extends AbstractCorfuTest {
+
+    private static final String TEST_HOST = "localhost";
```

Comment:
What if you need multiple servers? having static ports doesnt seem like a good idea. We can address that later though.